packages/common/package.json

```diff
@@ -23,6 +23,7 @@
     "class-validator": "^0.14.0"
   },
   "devDependencies": {
+    "@swc/core": "^1.3.86",
```

Comment:
Where is this dependency used?

Reply:
`tsup` uses this for `emitDecoratorMetadata` support